packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx

```diff
@@ -166,14 +176,52 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
       peopleButtonRef
     ]
   );
+
+  const locale = useLocale();
+
+  /* @conditional-compile-remove(spotlight) */
+  const callStrings = locale.strings.call;
+
+  /* @conditional-compile-remove(spotlight) */
```

Comment:
this file is huge and just getting bigger, and this code looks almost the same as the code in media gallery?

Consider moving this new logic into a hook, e.g.
```
/// useSpotlightHook.ts
const useSpotlightHook = () => {
  const locale = useLocale();
  ...
  return useMemo(() => ({
   startSpotlightWithPrompt,
   stopSpotlightWithPrompt
  }), [...]);
}
```